seedqreader.py

```diff
@@ -518,7 +519,7 @@ def list_available_cameras():
         available_cameras = []
         while True:
             cap = cv2.VideoCapture(index)
-            if not cap.isOpened():
+            if not cap.isOpened() and available_cameras:
```

Comment:
I'm not sure about this change, what about if the soft is run on a VM (no camera), in that case, the loop will be endless

Reply:
It will not, I was getting an error because I had no cameras, this made the error stop and the program execute, later I can plug the camera and the code still works

Reply:
This code `available_cameras.append(str(index))` on line 528 make this check to always work...

Reply:
The error before this fix was (with none usbcamera):
```
seedqreader.py", line 522, in list_available_cameras
    if (index - int(available_cameras[-1])) > 2:
                    ~~~~~~~~~~~~~~~~~^^^^
IndexError: list index out of range
```